lib/openhab/dsl/rules/builder.rb

```diff
@@ -1364,7 +1376,7 @@ def watch(path, glob: "*", for: %i[created deleted modified], attach: nil)
         # @return [true, false] True if rule should run on start, false otherwise.
         #
         def on_start?
-          @on_start.enabled
+          @on_start.enabled && start_level.nil?
```

Comment:
Having the old behaviour called on_load sounds reasonable. Everytime I see "on_start" I would immediately think system startlevel anyway. But that would cause a breaking change. To soften the blow, on_start (with system start) can default to level 100. WDYT?

Reply:
+1, I like it.